suite-native/storage/src/transforms/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { A, D, O } from '@mobily/ts-belt';
+
+import { DeviceRootState } from '@suite-common/wallet-core';
+
+export const selectDeviceStatesNotRemembered = (state: DeviceRootState) => {
+    return A.filterMap(state.device.devices, device =>
+        device.remember || !device.state ? O.None : O.Some(device.state),
+    );
+};
```

Comment:
nit: I would move this to `suite-native/device` package to selectors file, unless there is cyclic dependencies or similar reason..

Reply:
That would cause cyclic dependency and I beleive this selector won't be useful anywhere else.